spec/index.html

```diff
@@ -581,6 +609,186 @@ <h4>The <code>&lt;<dfn id="the-mapml-viewer-element">mapml-viewer</dfn>&gt;</cod
+            specifies the title of the output <code>layer</code> element. The 
+            default used will be, in order of preference, the 
+            <a href="https://www.rfc-editor.org/rfc/rfc7946#section-6.1">name</a> 
+            foreign member value, the <a href="https://www.rfc-editor.org/rfc/rfc7946#section-6.1">title</a>
+            foreign member value of the FeatureCollection or Feature object, or the fixed 
+            <code>'Layer'</code> DOMString value. 
+          </p>
+          <p>
+            The <a href="#dom-mapmlgenerationoptions-caption"><code>caption</code></a> option specifies the source
+            of the output <a href="#the-featurecaption-element"><code>featurecaption</code></a> value for 
+            each output <a href="#the-feature-element"><code>feature</code></a>.  
+            If <a href="#dom-mapmlgenerationoptions-caption"><code>caption</code></a>
+            is set to a DOMString value, that value will be sought as the name of
+            a <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">properties</a> 
+            member, the value of which property will be used as the output 
+            <a href="#the-featurecaption-element"><code>featurecaption</code></a>.  
+            If no such <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">properties</a> 
+            member exists, the fixed DOMString value itself will be used as
+            the output <a href="#the-featurecaption-element"><code>featurecaption</code></a> value.
+            
+            If <a href="#dom-mapmlgenerationoptions-caption"><code>caption</code></a> 
+            is set to a function value matching <a href="#dom-captioncallback"><code>CaptionCallback</code></a>,
+            that function will be called for each output <a href="#the-feature-element"><code>feature</code></a>,
+            and its return DOMString value will be used as the <a href="#the-feature-element"><code>feature</code></a>s' 
+            <a href="#the-featurecaption-element"><code>featurecaption</code></a> 
+            value.
+            
+            If no <a href="#dom-mapmlgenerationoptions-caption"><code>caption</code></a> value is provided, the 
+            default source of <a href="#the-featurecaption-element"><code>featurecaption</code></a> in output
+            <a href="#the-feature-element"><code>feature</code></a> elements will be, in order of preference,
+            the <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">Feature</a>
+            object's <a href="https://www.rfc-editor.org/rfc/rfc7946#section-3.2">id</a> member, it's <a href="https://www.rfc-editor.org/rfc/rfc7946#section-6.1">name</a> 
+            foreign member value, or its <a href="https://www.rfc-editor.org/rfc/rfc7946#section-6.1">title</a>
+            foreign member value.
+          </p>
+          <p>
+            The <a href=""><code>properties</code></a> option specifies how 
```

Comment:
Empty links, maybe can link to the GeoJson properties spec - https://www.rfc-editor.org/rfc/rfc7946#section-3.2